Cargo.toml

```diff
@@ -29,7 +29,7 @@ fixedbitset = "0.4.2"
 hashbrown = { version = ">=0.13, <0.15", features = ["rayon"] }
 indexmap = { version = ">=1.9, <3", features = ["rayon"] }
 num-traits = "0.2"
-numpy = "0.19.0"
+numpy = "0.20.0"
```

Comment:
Not this file but copy https://github.com/Qiskit/qiskit/pull/10893 for us esssentially

Reply:
Yeah, that sounds good. We definitely should do this for the 0.14.0 release.